src/extension/background-script/SteganographyService.ts

```diff
@@ -0,0 +1,47 @@
+import { encode, decode } from 'node-stego/es/dom'
+import { GrayscaleAlgorithm } from 'node-stego/es/grayscale'
+import { TransformAlgorithm } from 'node-stego/es/transform'
+import { OnlyRunInContext } from '@holoflows/kit/es'
+import { EncodeOptions, DecodeOptions } from 'node-stego/es/stego'
+
+OnlyRunInContext('background', 'SteganographyService')
+
+type WithPartial<T, K extends keyof T> = { [P in Exclude<keyof T, K>]?: T[P] | undefined } & { [P in K]: T[P] }
+
+const defaultOptions = {
+    size: 8,
+    narrow: 0,
+    copies: 3,
+    tolerance: 128,
+}
+
+export async function encodeImage(
+    { buffer }: Uint8Array,
```

Comment:
Question: Does Uint8Array can be transported across the context?

Reply:
Yes `Uint8Array` can persist transporting. I perfer use `ArrayBuffer` here but lost content after received in the other side.

Reply:
🤔Oh I found serialijse support TypedArray